aws-distro-opentelemetry-node-autoinstrumentation/src/patches/aws/services/bedrock.ts

```diff
@@ -182,10 +183,15 @@ export class BedrockServiceExtension implements ServiceExtension {
     };
   }
   responseHook(response: NormalizedResponse, span: Span, tracer: Tracer, config: AwsSdkInstrumentationConfig): void {
-    const guardrail_id = response.data[GUARDRAIL_ID];
+    const guardrailId = response.data[GUARDRAIL_ID];
+    const guardrailArn = response.data[GUARDRAIL_ARN];
 
-    if (guardrail_id) {
-      span.setAttribute(AWS_ATTRIBUTE_KEYS.AWS_BEDROCK_GUARDRAIL_ID, guardrail_id);
+    if (guardrailId) {
+      span.setAttribute(AWS_ATTRIBUTE_KEYS.AWS_BEDROCK_GUARDRAIL_ID, guardrailId);
+    }
+
+    if (guardrailArn) {
```

Comment:
The guardrailArn attribute gets populated in the span for Python but the Javascript one does not. 

![image](https://github.com/user-attachments/assets/4acb7e42-0c7b-47a9-9c34-bf26384316c9)